Source/UI/NewUI/PaymentForm/View/ExpiryDateView.swift

```diff
@@ -0,0 +1,129 @@
+import UIKit
+import Checkout
+
+protocol ExpiryDateViewDelegate: AnyObject {
+    func update(expiryDate: ExpiryDate)
+}
+
+final class ExpiryDateView: UIView {
```

Comment:
Should this not be public assuming this is a component which external dev will use to create their own ExpiryDateView, if then you also need to update the API spec as all the public facing APIs should be documented on our API spec.